seeq/addons/correlation/__main__.py

```diff
@@ -199,10 +199,16 @@ def cli_interface():
     user = args.username
     logging_attempts(user)
     seeq_url = args.seeq_url
+
+    try:
+        seeq_url_ = spy.session.public_url
+    except:
+        seeq_url_ = spy.client.host.replace('/api', "")
+
     if seeq_url is None:
-        seeq_url = input(f"\n Seeq base URL [{spy.client.host.split('/api')[0]}]: ")
+        seeq_url = input(f"\n Seeq base URL [{seeq_url_}]: ")
```

Comment:
Two comments here... 
1) If `spy.client.session.public_url` and `spy.client.session.private_url` work from the terminal of the SDL project and we check with a regex the URL is NOT an IP, then we can hide this step from the user (one less step during installation!)
2) If the any of the options we have to retrieve the URL (`spy.client.session.public_url`,  `spy.client.session.private_url` and `spy.client.host`) return an IP. Then, we ask the user for the Seeq base URL (as we currently do) BUT we also save that base URL in a hidden file of the SDL project (`~/.seeq/some_file`). Then when we need it in the `_seeq_worksheet_writer.py` we can retrieve it from the file. This is just an idea (kind of a hack) but could work, I think.

Reply:
1. Yes, 2. Ill have a look